src/components/Main/Schedule/NewServiceForm.tsx

```diff
@@ -32,30 +26,32 @@ const daysOfWeek = [
 ];
 
 export const NewServiceForm = ({ order, onSubmit, onClose }: NewServiceFormProps) => {
-  const [serviceName, setServiceName] = useState<TextFieldState>(
-    createTextFieldState(''),
+  const [
+    serviceName,
+    setServiceName,
+    setServiceNameError,
+    resetServiceNameError,
+  ] = useValidatedField('', 'Title must be not be blank and be under 32 characters long');
```

Comment:
Shorter and less ambiguous: 
`'Title must be filled and less than 32 characters',`

or:
`'Must have a title that is less than 32 characters',` 
and user assumes "must have a title" means > 0 chars.